### ISSUE TITLE
Report parameters require values, despite defaults

### ISSUE THREAD

**Issue by schmunk42** [Codeception/Codeception#4242]
#### What are you trying to achieve?

Creating reports.

#### What do you get instead?

No reports

```bash
codecept run -x optional --steps --html --xml --tap --json
```

Works with params

```bash
codecept run -x optional --steps --html=_report.html --xml=_report.xml --tap=_report.tap --json=_report.json
```
### Details

* Codeception version: 2.3.0
* PHP Version: 7.1.3
* Operating System: Alpine (Dokcer)
* Installation type: Composer

I thought this was working around `2.2.8`, but not 100% sure.

When running a single suite like so:

```
codecept run cli -x optional --steps --html --xml --tap --json
```

I get a file `cli` with json data?!

**Comment by DavertMik** [comment]
Sorry I couldn't reproduce your issue in 2.3.0

```
./codecept run cli -x optional --steps --html --xml --tap --json
```

At least for our internal tests

```
run with `-v` to get more info about skipped or incomplete tests
- HTML report generated in file:///home/davert/Codeception/tests/log/report.html
- XML report generated in file:///home/davert/Codeception/tests/log/report.xml
- TAP report generated in file:///home/davert/Codeception/tests/log/report.tap.log
- JSON report generated in file:///home/davert/Codeception/tests/log/report.json
```

**Comment by Lansoweb** [comment]
Hello! I'm having the same problem but with coverage.

Before upgrade from 2.2.10 to 2.3.2, I was able to generate coverage reports with:
```
codecept run --coverage-html
```
And the files from unit and api tests were saved inside tests/_output/coverage and tests/_output/api.remote.coverage, correctly.

But now, the files are not generated unless I specify a folder inside tests/_output, even ".":
```
codecept run --coverage-html .
```
And the files are inside tests/_output and tests/_output/api.remote.coverage.

Tried with phpunit 5.7.19 and 6.1.4.
